Automattic/jetpack
# Voice to Content: Update oscilloscope component with audio analyser

Part of #35547

## Proposed changes:

* Replaces the static image with a canvas linked to an audio analyser

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add a Voice to Content block
* Start recording
* Check that the oscilloscope reacts to your audio
* Pause the recording
* Check that the oscilloscope pauses
* Resume recording
* Check that the oscilloscope resumes as well
